comm/src/main/scala/coop/rchain/comm/transport/package.scala

```diff
@@ -0,0 +1,51 @@
+package coop.rchain.comm
+import java.nio.file.Path
+
+import coop.rchain.crypto.util.{CertificateHelper, CertificatePrinter}
+import coop.rchain.shared.Iso
+
+import scala.util.{Failure, Success, Try}
+
+case class Tls(
+    certificate: Path,
+    key: Path,
+    customCertificateLocation: Boolean,
+    customKeyLocation: Boolean,
+    secureRandomNonBlocking: Boolean
+)
+
+package object transport {
+  def generateSertificateIfAbsent[A: Iso[?, Tls]](a: A) = {
```

Comment:
This is effectful, please close over `F[_]`

Reply:
It wasn't previously. Where can I place the call of this function? `NodeRuntime.nodeProgram`?